src/auto/functions.rs

```diff
@@ -1218,8 +1208,7 @@ pub fn test_set_nonfatal_assertions() {
     }
 }
 
-pub fn test_skip<'a, P: Into<Option<&'a str>>>(msg: P) {
-    let msg = msg.into();
+pub fn test_skip(msg: Option<&str>) {
```

Comment:
Same for this for example. We generate lots of functions that are useless in Rust, not usable as-is from Rust, or are simply wrong :)